src/google-maps/README.md

```diff
@@ -0,0 +1,131 @@
+
+See the [example](#example) below or the [source](./google-map/google-map.ts) to read the API.
+
+## MapMarker
+
+The MapMarker component wraps the [google.maps.Marker class](https://developers.google.com/maps/documentation/javascript/reference/marker#Marker) from the Google Maps JavaScript API. The MapMarker component will display a marker on the map when it is a child of a GoogleMap component. Like the GoogleMap component, there is an `options` input as well as convenience inputs for `position`, `title`, `label`, and `clickable`, and supports all google.maps.Marker events as outputs.
+
+See the [example](#example) below or the [source](./map-marker/map-marker.ts) to read the API.
+
+## MapInfoWindow
+
+The MapInfoWindow component wraps the [google.maps.InfoWindow class](https://developers.google.com/maps/documentation/javascript/reference/info-window#InfoWindow) from the Google Maps JavaScript API. The Info Window has a `options` input as well as a convenience `position` input. Content for the MapInfoWindow is the inner HTML of the component, and will keep the structure and css styling of any content that is put there when it is displayed as an InfoWindow on the map.
+
+To display the MapInfoWindow, it must be a child of a GoogleMap component, and it must have its `open` method called, so a reference to MapInfoWindow will need to be loaded using the [ViewChild decorator](https://angular.io/api/core/ViewChild). The `open` method accepts an MapMarker as an optional input, if you want to anchor the MapInfoWindow to a MapMarker.
+
+See the [example](#example) below or the [source](./map-info-window/map-info-window.ts) to read the API.
+
+## Other Google Maps features
+
+Wrappers for other Google Maps features such as Polygons or overlays are not yet implemented, but can be used by directly accessing the google.maps.Map object on the GoogleMap component, which is sat as the `_googleMap` property.
```

Comment:
Maybe we should remove the underscore from `_googleMap` if it's intended to be used publicly?

Reply:
Hmmmmmmmm, good point. I don't really want this to be made public, but since it's a gigantic API that isn't yet fully replicated by these components, I thought users would sometimes need to have access to the underlying google.maps.Map object. I have removed this section for now, and will add it back in when I make googleMap public in a separate pr.